_content/en/03-dont/05-do-not-hoard-masks.md

```diff
@@ -1,4 +0,0 @@
-{:.dont}
-## Do not hoard masks
```

Comment:
The word "hoard" is doing a lot of work here, but is vague. What constitutes hoarding? What about buying one respirator for each member of the family? Is that hoarding? What if you buy 60 to wear on the way to work on the train, and you expect to use 1 per day and have to wear them for 2 months? Would that qualify?